### PR TITLE
Do not fail migration on missing biodi:Edge#source

### LINKED ISSUE #18: Migration does not work for some dmn diagrams
__Steps to Reproduce__

1. Try to migrate this diagram [diagram11.dmn.zip](https://github.com/bpmn-io/dmn-migrate/files/4292980/diagram11.dmn.zip) using the migration tool
2. The migration fails with the following error
image

__Environment__

- OS: Mac OSX
- Library version: "dmn-js": "^8.0.1"

### PR DESCRIPTION
**Pull request by nikku**
Closes #18

### CODE REVIEW

File: src/migrateDiagram.js [hunk 1]
```diff
@@ -320,15 +320,20 @@ function migrateDI(definitions, moddle) {
       }
 
       if (is(extensionElement, 'biodi:Edge')) {
+        const dmnElementRef = getDMNElementRef(semantic, extensionElement.get('source'));
+
+        // DMN element does exist
```
Comment (philippfromme) [inline src/migrateDiagram.js]: Does or doesn't?

Comment (nikku) [reply src/migrateDiagram.js]: Let's leave this typo in here and see how long it takes until someone notices.

Comment (philippfromme) [reply src/migrateDiagram.js]: Lol.

Comment (nikku) [reply src/migrateDiagram.js]: I've updated the PR as I'm not sure what _Lol._ refers to.

**Pull request merged**